src/Pim/Bundle/CatalogBundle/Resources/config/validation/product.yml

```diff
@@ -12,6 +15,14 @@ Pim\Component\Catalog\Model\Product:
             groups:
                 - Default
                 - pim_catalog_variant_group
+    properties:
+        identifier:
+            - NotBlank: ~
+            - Regex:
+                pattern: '/^((?![,;]).)*$/'
```

Comment:
I tried but is did not work (but I am kind of a noob in regex, so maybe I didn't write it right).